.travis.yml

```diff
@@ -152,6 +156,10 @@ jobs:
       script: ./tools/travis_controller.sh tests
       env: PROFILE="-Dhadoop.version=2.4.1 -Pskip-hive-tests"
       name: tests - hadoop 2.4.1
+    - if: type = cron
+      script: ./tools/travis_controller.sh scheduler_ng
+      env: PROFILE="-Dhadoop.version=2.4.1 -Dscheduler-ng -Pskip-hive-tests"
```

Comment:
Not sure if it's needed to test the new scheduler against different hadoop versions. We don't have direct dependencies to hadoop. Same holds for scala 2.12. However, JDK11 might be ok.

Reply:
Agreed that the scheduler has nothing direct related with the hadoop version or scale version.
Will remove those two.

Reply:
I don't think we need the jdk 11 build. So long as you don't do any funky class-loading things I doubt you'll break something that isn't caught by other tests.

Reply:
Ok. We changed nothing on class loading in the new scheduler.

Reply:
done.